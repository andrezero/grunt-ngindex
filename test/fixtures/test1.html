<!DOCTYPE html>
<html>

<head>

    <title><%= pkg.name %></title>
    <% if ('undefined' !== typeof cssFiles) { cssFiles.forEach( function ( file ) { %>
    <link rel="stylesheet" type="text/css" href="/<%= file %>" /><% }); } %>

</head>

<body>

    <header>
        <h1><%= pkg.name %></h1>
        <h2><%= foo.bar %></h2>
        <h2><%= foo.baz %></h2>
    </header>

    <% if ('undefined' !== typeof jsFiles) { jsFiles.forEach( function ( file ) { %>
    <script type="text/javascript" charset="utf-8" src="/<%= file %>"></script><% }); } %>

</body>

</html>
