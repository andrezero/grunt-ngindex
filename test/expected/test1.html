<!DOCTYPE html>
<html>

<head>

    <title>grunt-ngindex</title>
    
    <link rel="stylesheet" type="text/css" href="/fixtures/style1.css" />

</head>

<body>

    <header>
        <h1>grunt-ngindex</h1>
        <h2>baz</h2>
        <h2>qux</h2>
    </header>

    
    <script type="text/javascript" charset="utf-8" src="/fixtures/script1.js"></script>

</body>

</html>
